pkg/ccl/backupccl/backup_processor.go

```diff
@@ -247,9 +239,9 @@ type spanAndTime struct {
 	lastTried  time.Time
 }
 
-type returnedSST struct {
-	f              BackupManifest_File
-	sst            []byte
+type returnedSpans struct {
+	partialFile    BackupManifest_File
+	spanDataAsSST  []byte
```

Comment:
Go generally discourages repeated word or "stuttering" names, e.g. preferring "user.Email" over "user.UserEmail", and letting the containing type/package/etc speak for itself, so in this case I think it'd be "returnedSpan.dataSST" since it is the data field _of_ the returnedSpan.